esapi/esapi/Esys_PolicyCommandCode.c

```diff
@@ -55,10 +55,11 @@ static void store_input_parameters (
  * parameters is allocated by the function implementation.
  *
  * @param[in,out] esysContext The ESYS_CONTEXT.
+ * @param[in] policySession Input handle of type ESYS_TR for
+ *     object with handle type TPMI_SH_POLICY.
```

Comment:
... of type ESYS_TR

Reply:
what's the change you're asking for here @tstruk?

Reply:
Never mind. I just read the second part of the sentence and missed the ... Input handle of type ESYS_TR for ...